tests/PathTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace OpenCFP\Test;
+
+use OpenCFP\Environment;
+use OpenCFP\Path;
+
+/**
+ * @covers \OpenCFP\Path
+ */
+class PathTest extends BaseTestCase
```

Comment:
Do we really need to extend the `BaseTestCase` here? That is, are we using any of the functionality provided by it?

Also, what do you think about marking the test as `final`?